plugins/woocommerce/client/blocks/assets/js/blocks/add-to-cart-with-options/frontend.ts

```diff
@@ -448,21 +449,11 @@ const { actions, state } = store<
 
 				if ( newValue !== currentValue ) {
 					actions.setQuantity( newValue );
+
 					inputElement.value = newValue.toString();
 					dispatchChangeEvent( inputElement );
 				}
 			},
-			handleQuantityInput: (
-				event: HTMLElementEvent< HTMLInputElement >
-			) => {
-				const inputData = getInputData( event );
-				if ( ! inputData ) {
-					return;
-				}
-				const { currentValue } = inputData;
-
-				actions.setQuantity( currentValue );
-			},
```

Comment:
Now, we always call `setQuantity()` in `handleQuantityBlur()`, so there is no need to call it also on input events.